### ISSUE TITLE
Slowness and hangs related to "Connecting to the Parity Secure API"

### ISSUE THREAD

**Issue by knocte** [ethcore/parity#3811]
I've compiled parity myself (master branch, labeled as 1.5.0-beta), and it seems to work ok on the MainNet.

However, if I run it with flag `--chain=classic`, I then cannot connect to the UI (in http://localhost:8080/) anymore. When I do that, it tries to redirect me to http://127.0.0.1:8180/#/accounts but it stays there with the message "Connecting to the Parity Secure API" and I cannot do anything else.

**Comment by knocte** [comment]
I spoke too soon, I guess after some minutes of running in the background, the UI now prompted me the message about running `parity signer new-token`, then I did, and grabbed the token and pasted it in the textbox, but nothing happens, I press enter and the dialog (where it asks me for the token) stays there.

**Comment by knocte** [comment]
If I click the link `http://127.0.0.1:8180/#/auth?token=foo` instead of pasting the token in the textbox, it goes again to a dialog that says "Connecting to the Parity Secure API", and it hangs there.

**Comment by knocte** [comment]
I used this guide: https://stacktile.io/org/ethcore/workflows/12  So yes, I used the `--release` flag.

**Comment by YamashitaRen** [comment]
Same here.
I launch parity, go on the webui, get the "run parity signer new-token" message so I do it.
I copy-paste or write the key. Once I write the last character, the code disapear and I get the original "blank" field again.

**Comment by knocte** [comment]
I tried a couple of things today:

- Reverted back to use 1.4.6 instead of 1.5.0-beta. This didn't help, I was still having slowness and issues with the tokens.

Then I tried removing the cookies in my browser because everytime I wanted to connect to http://localhost:8080, I could see this in the log:

```
2016-12-18 17:57:13  Received old authentication request. (1482055033 vs 1482054973)
2016-12-18 17:57:13  Unauthorized connection to Signer API blocked.
```

However, that didn't help.

But then I tried with other browser: chromium instead of firefox. And it worked! I'm inside the interface now. (This is a crappy workaround for sure though.)

**Comment by zoltanmaric** [comment]
I'm running into the same issue today. I'm running parity with the `tobalaba` blockchain. I've followed instructions [here](https://energyweb.atlassian.net/wiki/spaces/EWF/pages/72974337/How+to+get+startet+with+Tobalaba), and the UI just can't connect in Firefox, whereas in Chromium it works fine. I've also tried with `--jsonrpc-interface all --jsonrpc-cors all --ui-interface all --ws-interface all --ws-origins all` and `--unsafe-expose`, but it didn't help.

Checking the console in Firefox, I see I get a `403` for the `websocket` request on the `GET /` endpoint.
